test/optimizer/index_nested_loops_join_test.cpp

```diff
@@ -46,7 +46,7 @@ struct IdxJoinTest : public TerrierTest {
     TERRIER_ASSERT(result.type_ == trafficcop::ResultType::COMPLETE, "Bind should have succeeded");
 
     auto plan = tcop_->OptimizeBoundQuery(common::ManagedPointer(&context_), stmt.ParseResult());
-    if (qtype >= network::QueryType::QUERY_CREATE_TABLE) {
+    if (qtype >= network::QueryType::QUERY_CREATE_TABLE && qtype != network::QueryType::QUERY_CREATE_INDEX) {
```

Comment:
Nit: It might be better to have a descriptive function / this seems a bit hacky. See NetworkUtil::CreateQueryType.

Reply:
per our discussion, we agreed to leave it for now